indico/modules/events/editing/client/js/editing/EditableSubmissionButton.jsx

```diff
@@ -81,6 +85,12 @@ export default function EditableSubmissionButton({
                       contrib_id: contributionId,
                       type: currentType,
                     })}
+                    uploadExistingURL={apiUploadExistingURL({
+                      confId: eventId,
+                      contrib_id: contributionId,
```

Comment:
```suggestion
                      contribId: contributionId,
```

Reply:
Those are parameters to build an URL. They must be in snake_case because that's how they are defined in the blueprint..

Reply:
Yes, unfortunately, there are way too many references to change. Perhaps in a separate task.

Reply:
We do not want to change this because that would introduce camelCase on the Python side ;)

Reply:
Yes, I was thinking about `confId` to `conf_id`.

Reply:
That would be nice in principle - except that the standard for names in JS is camelCase...

The `snakifyKeys` util would work, but for less widespread names using it would have the disadvantage that searching for the param name using one style would not find occurrences of it in the other style. With the explicit mapping you at least find the place with the mapping basically telling you that there's something else to search for..